test/test_sympy_utils.py

```diff
@@ -258,7 +257,13 @@ def test_interp(self, fn):
         if fn in ("div", "truncdiv", "minimum", "maximum", "mod"):
             return
 
-        from sympy.abc import x, y
+        is_integer = None
+        if fn == "pow_by_natural":
+            is_integer = True
+
+        x = sympy.Dummy('x', integer=is_integer)
+        y = sympy.Dummy('y', integer=is_integer)
```

Comment:
Dummy is probably not necessary here, I was just worried about accidentally mixing up symbols between tests